src/sbt-test/sbt-coffeescript-plugin/error-output/src/main/assets/a.coffee

```diff
@@ -1 +1 @@
-*(U$@QNJH#R)
```

Comment:
Needed to change this because it crashed later versions of CoffeeScript!

https://github.com/jashkenas/coffeescript/issues/4575